docs/en/concept/schema-feature.md

```diff
@@ -24,6 +26,14 @@ schema = {
 }
 ```
 
+### table
+
+The table full name of the table identifier which the schema belongs to, it contains database, schema, table name. e.g. `database.schema.table`, `database.table`.
```

Comment:
We cannot support `schema.table` yet. 
Since if user write `a.b`, we cannot know whather the given `a` is schema or database. And the logic is same with current code in TablePath::of(String fullName), the database and table is mandatory but schema is chooseable.

Reply:
I update this PR, we can support config `schema.table` and `table` in config.